src/ui/templates/partials/insurance/broker-details.njk

```diff
@@ -0,0 +1,13 @@
+{% set detailsHtml %}
+  {% include "partials/insurance/broker-description.njk" %}
+{% endset %}
+
+<div class="govuk-grid-column-three-quarters-from-desktop">
```

Comment:
Does this need a wrapping `<div class="govuk-grid-row">`?

Reply:
its working well without the div